### ISSUE TITLE
Suggestion: Hotkey to freeze preview UI to allow hovering over preview items to view their usual hover tooltip

### ISSUE THREAD

**Issue by Qwertylex** [MisterPeModder/ShulkerBoxTooltip#48]
Would it be possible to add a feature where you could press a hotkey to freeze the preview UI then hover over items in the preview UI to see the item's usual hover UI (such as item name, enchantments, lore., etc)?

**Comment by MisterPeModder** [comment]
While it would be possible, I think it would look pretty messy, considering the preview will be located *inside* the item's tooltip starting with Minecraft 1.17.

**Comment by Qwertylex** [comment]
ah well the server I play on will probably be stuck on 1.16 for the next two years so..

**Comment by Hi-ImKyle** [comment]
+1 for this. I'd try to implement it myself but I can't open the project in IDEA without gradle getting upset it cant find loom so..

>considering the preview will be located inside the item's tooltip starting with Minecraft 1.17.

Maybe add an option to not have this? I much prefer the way it is at the moment...

**Comment by MisterPeModder** [comment]
The vanilla-style placement is the new default in 1.17, but there will be a setting in the config to revert it back to the way it is before 1.17

**Comment by Hi-ImKyle** [comment]
Well either way, even if it does look messy how would one go about creating this? I've tried cloning the repo and opening it in IDEA but like I said gradle can't find loom

```
Could not find Jankson-Fabric-3.0.1+j1.2.0@net.fabricmc.yarn.1_16_5.1.16.5+build.4-v2.jar (io.github.cottonmc:Jankson-Fabric:3.0.1+j1.2.0@net.fabricmc.yarn.1_16_5.1.16.5+build.4-v2).
Searched in the following locations:
    file:/C:/Users/Kyle/.gradle/caches/fabric-loom/20w51a-mapped-net.fabricmc.yarn-20w51a+build.5-v2/Jankson-Fabric-3.0.1+j1.2.0@net.fabricmc.yarn.1_16_5.1.16.5+build.4-v2.jar
    file:/C:/Users/Kyle/.gradle/caches/fabric-loom/20w51a-mapped-net.fabricmc.yarn-20w51a+build.5-v2/Jankson-Fabric.jar
```

Normally I'd expect it to fetch it and be fine..

**Comment by MisterPeModder** [comment]
I don't know about setting up the project in IDEA, but it looks like a cache issue.
Try running `.\gradlew.bat cleanLoomMappings cleanLoomBinaries` maybe ?

**Comment by Hi-ImKyle** [comment]
Unfortunately no luck in which case I will just have to +1 this request regardless of how "messy" it may be, it's a feature people will probably like nonetheless.

**Comment by NitsuSaiNeko** [comment]
Using a scroll wheel as a way to highlight an item in preview might also be a good alternative to freezing UI and point at it.

**Comment by MisterPeModder** [comment]
![image](https://github.com/MisterPeModder/ShulkerBoxTooltip/assets/7838479/f473e062-20b5-4d42-bed3-e62257abd3d8)

It's finally ready to go! In the next releases for 1.20 and 1.19.4, you will be able to lock the tooltip using a key (left control by default) and hover inner items in the preview.

**Comment by Hi-ImKyle** [comment]
I'm probably not going past 1.19.3 so I'm hoping I can build it for that, but looks great. Glad you could implement it 👍

**Comment by MisterPeModder** [comment]
The feature is available for 1.19.4!
